### ISSUE TITLE
Builds failing because of pep8 version

### ISSUE THREAD

**Issue by dcolligan** [ga4gh/server#215]
We need to figure out which version of flake8 has an appropriate version of pep8 and hardcode it in the requirements.txt file.

The current version of flake8 uses pep8 1.6.2.  We need a version of pep8 above or equal to 1.5.7 and less than 1.6.

```
$ flake8 tests ga4gh scripts
Traceback (most recent call last):
File "/home/travis/virtualenv/python2.7.9/bin/flake8", line 11, in <module>
  sys.exit(main())
File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/flake8/   main.py", line 24, in main
  flake8_style = get_style_guide(parse_argv=True, config_file=DEFAULT_CONFIG)   File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/flake8/   engine.py", line 110, in get_style_guide
  kwargs['parser'], options_hooks = get_parser()
File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/flake8/   engine.py", line 48, in get_parser
  (extensions, parser_hooks, options_hooks, ignored) = _register_extensions()
File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/flake8/   engine.py", line 32, in _register_extensions
  checker = entry.load()
File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/          pkg_resources/__init__.py", line 2309, in load
  self.require(*args, **kwargs)
File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/          pkg_resources/__init__.py", line 2326, in require
  items = working_set.resolve(reqs, env, installer)
File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/          pkg_resources/__init__.py", line 810, in resolve
  raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.VersionConflict: (pep8 1.6.2 (/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages), Requirement.parse('pep8<1.6,>=1.5.7'))
The command "flake8 tests ga4gh scripts" exited with 1.
```

**Comment by dcolligan** [comment]
Er, actually, pep8 is set in our requirements.txt file itself.  So we can just hardcode it there.

**Comment by dcolligan** [comment]
Fixed in d9db9d0ae3e47b5ecd5a0d01731b7b3384829226
